### DISCUSSION TITLE
Should webmentions be automatically resend after updating a post?

### DISCUSSION THREAD

**Discussion by kandr3s** [jlelse/GoBlog#27] [Q&A] [answered]
I noticed GoBlog resends Webmentions whenever a post or page gets updated. I was wondering if this is the right approach as there might be pages that need to be updated constantly and that would spam any of the people mentioned in the page. What do you think of making outgoing webmentions manual after they have been sent already once?

**Comment by jlelse** [comment, marked as answer]
A button "Resend Webmentions" (similar to "Regnerate TTS") sounds like a good option. Accompanied by a setting "Resend webmentions after post updates".

What is already possible is to set the parameter `webmention: "false"` when updating the post. ([source](https://github.com/jlelse/GoBlog/blob/d186f2e9569200e4159ea9dc0d4d87ef0fe1476a/webmentionSending.go#L31))
